hub_module/modules/image/Colorization/User_guided_colorization/module.py

```diff
@@ -0,0 +1,212 @@
+# copyright (c) 2020 PaddlePaddle Authors. All Rights Reserve.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import paddle
+import paddle.fluid as fluid
+import numpy
+import paddle.nn as nn
+from paddle.nn import Conv2d, ConvTranspose2d
+from paddlehub.module.module import moduleinfo
+
+from paddlehub.process.transforms import *
+from paddlehub.module.cv_module import ImageColorizeModule
+
+
+@moduleinfo(
+    name="User_guided_colorization",
+    type="CV/image_editing",
+    author="baidu-vis",
+    author_email="",
+    summary="User_guided_colorization is a image colorization model, this module is trained with ILSVRC2012 dataset.",
+    version="1.0.0",
+    meta=ImageColorizeModule)
+class Userguidedcolorization(nn.Layer):
+    """Userguidedcolorization, see https://github.com/haoyuying/colorization-pytorch
+    Args:
+        use_tanh (bool): Whether to use tanh as final activation function.
```

Comment:
use_tanh (bool): Whether to use tanh as final activation function, default is True.